### DISCUSSION TITLE
Feature: SlowMode for chat

### DISCUSSION THREAD

**Discussion by Meldiron** [Zrips/CMI#1325] [Feature Requests]
Commands:
**/cmi slowchat start (time)**
Example: /clo slowchat start 5m
Will enable slow mode and people will have ti wait 5 minutes to be able to write next message

**/cmi slowchat stop**
Will stop slowmode chat so people no longer need to wait

Permissions:
1. permission to use start/stop commands
2. permission to bypass slow mode (can write frequently in slow mode)

**Comment by SenPr** [comment]
Please consider this, or adding some form of chat delay in chatfilter

**Comment by mrfloris** [comment]
During flood attacks this would be kinda handy, then flood bots will fail, while regular players can still chat a bit. if this is a perm, then we can slowmode default group and remove it from the regulars group

**Comment by MILKv2** [comment]
hey i still want delay in chat in 2025, please add it so user with permission or group

**Comment by CloudeLecaw** [comment]
i know its not what exacly is wanted and asked here
but may in the time try out this plugin

https://www.spigotmc.org/resources/zslow.115755/

It comes with a setbale cooldown for chat
allows bypass permission and black and whitelistings of player to bypass the cooldown

and and best...it works fine together with cmi chat :-)


Still i also would like to see a Slowdown featur. may as suggested as part of the Chatfilter because that may suits most to the thing what a slowdown is for. to keep a chat cleaner...
